### ISSUE TITLE
[Bug] [2.1.3] Crash When Using Rift Mover

### ISSUE THREAD

**Issue by FoxMcloud5655** [TheCodex6824/ThaumicAugmentation#244]
Running Thaumic Augmentation version 2.1.3, one of my players got a crash when attempting to move a rift.  Relevent parts of the crash report are as follows:
```
java.lang.NullPointerException
    at thecodex6824.thaumicaugmentation.common.tile.TileRiftMoverInput.update(TileRiftMoverInput.java:197)
```
```
-- Block entity being ticked --
  Name: thaumicaugmentation:rift_mover_input // thecodex6824.thaumicaugmentation.common.tile.TileRiftMoverInput
  Block type: ID #5293 (tile.thaumicaugmentation.rift_mover_input // thecodex6824.thaumicaugmentation.common.block.BlockRiftMoverInput // thaumicaugmentation:rift_mover_input)
  Block data value: 0 / 0x0 / 0b0000
  Block location: World: (932,62,3149), Chunk: (at 4,3,13 in 58,196; contains blocks 928,0,3136 to 943,255,3151), Region: (1,6; contains chunks 32,192 to 63,223, blocks 512,0,3072 to 1023,255,3583)
  Actual block type: ID #5293 (tile.thaumicaugmentation.rift_mover_input // thecodex6824.thaumicaugmentation.common.block.BlockRiftMoverInput // thaumicaugmentation:rift_mover_input)
  Actual block data value: 0 / 0x0 / 0b0000
  Block Entity NBT: {size:0,seed:0,stab:0.0f,x:932,y:62,z:3149,operating:1b,id:"thaumicaugmentation:rift_mover_input"}
```
If any more information is desired, I can provide the full crash report, but I think this is enough to go on.

**Comment by Vapaman** [comment]
Possibly a dublicate of https://github.com/TheCodex6824/ThaumicAugmentation/issues/226; fixed in 2.1.4

**Comment by DoomRater** [comment]
But also feel free to reopen if 2.1.4 doesn't fix it.
